plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py

```diff
@@ -23,12 +23,12 @@ class RedisConf:
 
 @dataclass
 class EnqueueConf:
-    # maximum runtime of the job before it's interrupted and marked as failed (in sec)
-    job_timeout: Optional[int] = None
-    # maximum queued time before the job before is discarded (in sec)
-    ttl: Optional[int] = None
-    # how long successful jobs and their results are kept (in sec), default: 10 days
-    result_ttl: int = 864000
+    # maximum runtime of the job before it's interrupted (in sec), default: 100 days
+    job_timeout: int = 8640000
```

Comment:
By interrupted you mean "murdered with a sharp signal"?

100 days seems large enough to indicate that you are actually just trying to disable it. Is there no way to disable it in Redis?

Reply:
Exactly, the process gets killed and the job moves to the failed queue. 

No, it's not possible to disable it in Python RQ -- I was tricked by this as well. While it's possible to pass `job_timeout=None`, this will internally default to a `180` second timeout. I think it's much more preferable to default to not enforcing time limits which this PR effectively does

Reply:
You think 100 days as infinity, but it's not.
I actually had ML jobs that took over 3 months to complete.

Can you file an issue against Redis queue to allow them to accept no-timeout?

Reply:
Found a way around, `-1` should work for inf `job_timeout`. Does that sound good?

Reply:
Much better.
Be sure to add a comment in the dataclass.

Reply:
Looking into it, I figured that instead of passing integers, strings with units can be passed, e.g., `24h`. I think this is much easier to use and understand than using integers with special meanings (such as -1 for infinite, and expressing all timeouts by converting to seconds). 

I have updated the code in the latest commit to use 1000 days (`1000d`) as default timeout for all four values.

Reply:
I still prefer -1 for infinite or another constant if they support it.
if they take a string or an int, use Any as the type and do validation that i's a string or an int in the code itself. (as of now OmegaConf does not support [Union[int,str]](url)).

Reply:
In the latest commit, types are `Optional[str]` defaulting to `None`. Internally, `None` is converted such that it means no limit (the internal conversion necessary to archive that differs between the values, but it's just internals this way that a user doesn't need to worry about).

Reply:
In addition, I updated the example to show how a default timeout value can be overridden